### PR TITLE
Add a :city attribute to People

### PR DESCRIPTION
**Pull request by dylanwolff**
Pretty self-explanatory.

### CODE REVIEW

File: apps/main/lib/main/views/pages/about.rb [hunk 1]
```diff
@@ -17,6 +17,7 @@ class About < Main::View
         def locals(options = {})
           super.merge(
             people: Decorators::PublicPerson.decorate(people.for_about_page),
+            number_of_cities: people.select(:city).map { |p| p[:city] }.uniq.length
```
Comment (timriley) [inline apps/main/lib/main/views/pages/about.rb]: @dylanwolff isn't this still doing an SQL query? I'm suggesting we actually operate on the people array we're fetching from the repo just above.

Comment (dylanwolff) [inline apps/main/lib/main/views/pages/about.rb]: Is this fine @timriley?:
```
def locals(options = {})
  super.merge(
    people: Decorators::PublicPerson.decorate(people.for_about_page),
    number_of_cities: people.for_about_page.map { |p| p[:city] }.uniq.length
  )
end
```
With this approach I needed to explicitly add my "sample" person to the `/about` page in order to display the correct city count. Took some time to figure that out.

File: apps/main/lib/main/views/pages/about.rb [hunk 2]
```diff
@@ -17,7 +17,7 @@ class About < Main::View
         def locals(options = {})
           super.merge(
             people: Decorators::PublicPerson.decorate(people.for_about_page),
-            number_of_cities: people.select(:city).map { |p| p[:city] }.uniq.length
+            number_of_cities: people.for_about_page.map { |p| p[:city] }.uniq.length
```
Comment (dylanwolff) [inline apps/main/lib/main/views/pages/about.rb]: I should have pinged you on this one yesterday @timriley. Does this look OK?

Comment (timriley) [inline apps/main/lib/main/views/pages/about.rb]: I think this'll still run the query twice. What I think you want to do is to assign `people.for_about_page.to_a` to a var up above `super.merge` and then refer to it for both of the keys inside this hash.

Comment (dylanwolff) [inline apps/main/lib/main/views/pages/about.rb]: Ahh, roger. I misunderstood. Thanks.

**Comment by dylanwolff** [comment]
Merging 🚚

**Pull request merged**